### ISSUE TITLE
scCryptolType: unsupported type sort 0

### ISSUE THREAD

**Issue by msaaltink** [GaloisInc/saw-script#1248]
Executing

    let bv64 = parse_core "bitvector 32";

results in

    [17:36:02.544] You have encountered a bug in cryptol-saw-core's implementation.
    *** Please create an issue at https://github.com/GaloisInc/saw-core/issues

    %< --------------------------------------------------- 
      Revision:  cff300fce2caaf4be678bfdab8c8e3ade93d020f
      Branch:    HEAD
      Location:  scCryptolType
      Message:   scCryptolType: unsupported type sort 0
    CallStack (from HasCallStack):
      panic, called at src/Verifier/SAW/Cryptol/Panic.hs:13:9 in cryptol-saw-core-0.1-7NBapC0zhks5IhYxksTPfp:Verifier.SAW.Cryptol.Panic
      panic, called at src/Verifier/SAW/Cryptol.hs:1400:19 in cryptol-saw-core-0.1-7NBapC0zhks5IhYxksTPfp:Verifier.SAW.Cryptol
    %< ---------------------------------------------------

**Comment by RyanGlScott** [comment]
I believe this is fixed now? On a recent checkout of `master`, neither of the following appear to crash:

```
sawscript> parse_core "Vec 32 Bool"
[15:37:21.828] Prelude.Vec 32 Prelude.Bool
sawscript> parse_core "4"
[15:37:23.726] 4
```

(I'm assuming that `bitvector n` is an alias for `Vec n Bool`.)

**Comment by robdockins** [comment]
Yes, this was fixed via #1336.  It's probably worthwhile to keep this ticket open until we add a test case.
